libgraph/src/analytics/leiden_clustering/leiden_clustering.cpp

```diff
@@ -46,8 +46,7 @@ struct LeidenClusteringImplementation
   using CommTy = LeidenCommunityType<EdgeWeightType>;
   using CommunityArray = katana::NUMAArray<CommTy>;
 
-  using Graph = katana::TypedPropertyGraphView<
-      katana::PropertyGraphViews::Undirected, NodeData, EdgeData>;
+  using Graph = katana::TypedPropertyGraphView<ViewTy, NodeData, EdgeData>;
```

Comment:
Define `struct GraphTypes` similar to what is in Louvain to make it more readable.